core/net/addr.odin

```diff
@@ -34,11 +34,8 @@ import "core:fmt"
 	If `allow_non_decimal` is true, `aton` is told each component must be decimal and max 255.
```

Comment:
I renamed that variable but didn't update the comment to also mean the opposite.